maps/site53/items/clothing/solgov-head.dm

```diff
@@ -7,30 +7,30 @@
 
 //Utility
 /obj/item/clothing/head/soft/solgov
```

Comment:
replace type name
(like for all of them)

Reply:
fixed